src/components/autocomplete/js/autocompleteController.js

```diff
@@ -241,16 +242,36 @@ function MdAutocompleteCtrl ($scope, $element, $mdUtil, $mdConstant, $mdTheming,
       if (elements) {
         $mdUtil.nextTick(function () {
           $mdUtil.disableScrollAround(elements.ul);
+          revertWrapScrollBlocking = disableElementScrollEvents(angular.element(elements.wrap));
         }, false, $scope);
       }
     } else if (hidden && !oldHidden) {
       $mdUtil.nextTick(function () {
         $mdUtil.enableScrolling();
+        if (revertWrapScrollBlocking) revertWrapScrollBlocking();
```

Comment:
I don't that's necessary, because the `off` events are bound to an unique event function, but I will update it, because it seems to be logical.